geometry/perimeter/java/Perimeter.java

```diff
@@ -68,6 +68,7 @@
      */
     double trapezoid(double height, double base1, double base2)
     {
-    	return height * (base1 + base2)/2;
+    	return 2*Math.sqrt(height*height+((base1-base2)/2*(base1-base2)/2))+base1+base2;
```

Comment:
Where did you get this formula from? Also, please space everything out more. rename file to trapezoid_perimeter.java

Reply:
Did you mean renaming the file name or function name (I did not use a separate file)- The formula is for an isosceles trapezoid and the formula is (slightly) modified from  http://mathworld.wolfram.com/IsoscelesTrapezoid.html